adcm_aio_client/core/objects/cm.py

```diff
@@ -115,8 +115,8 @@ def signature_status(self: Self) -> Literal["invalid", "valid", "absent"]:
     def _type(self: Self) -> Literal["cluster", "provider"]:
         return self._data["mainPrototype"]["type"]
 
-    @property
-    def license(self: Self) -> License:
+    @async_cached_property
+    async def license(self: Self) -> License:
```

Comment:
no await in here, something forgotten?

or just for similarity with Service API?

Reply:
yes, for .license() calls consistency